docs/source/markdown/podman-pod-create.1.md

```diff
@@ -4,13 +4,13 @@
 podman\-pod\-create - Create a new pod
 
 ## SYNOPSIS
-**podman pod create** [*options*]
+**podman pod create** [*options*] [*name*]
 
 ## DESCRIPTION
 
 Creates an empty pod, or unit of multiple containers, and prepares it to have
-containers added to it. The pod id is printed to STDOUT. You can then use
-**podman create --pod `<pod_id|pod_name>` ...** to add containers to the pod, and
+containers added to it. The pod can be created with a specific name, if a name is not given a random name is generated. The pod id is printed to STDOUT. You can
```

Comment:
I would prefer a semicolon or even a period here, not a comma: `...a specific name. If a name is not...`

(minor nit, but I think you have to re-push anyway to fix a test)